chunks/tool-nginx/Dockerfile

```diff
@@ -6,14 +6,16 @@ USER root
 # Dazzle does not rebuild a layer until one of its lines are changed. Increase this counter to rebuild this layer.
 ENV TRIGGER_REBUILD=1
 
-RUN install-packages \
+RUN for _ppa in 'ppa:ondrej/php' 'ppa:ondrej/apache2' 'ppa:ondrej/nginx-mainline'; do add-apt-repository -y "$_ppa"; done \
+    && install-packages \
```

Comment:
I am bit confused about the PR. The associated issue says not to use `ppa:ondrej/php` but that is still being added here. And +1 to the suggestion of specifying the version of the PHP.

Reply:
Perhaps we create multi-version docker images?

Reply:
@axonasif yes, Dazzle variants make sense for PHP 7.4 and 8.0, which'll result in separate images being published to Docker Hub. 

Also, `ppa:ondrej/php` should be removed, I think. I am going to flip this PR back to draft for now. 

Is this something you would still like to own? If yes, what do you think of the new scripts to build and test chunks and combinations?

Reply:
Hey @princerachit and @kylos101 👋 
I haven't been able to find any php8.0(or X) package in the official ubuntu and/or debian repositories.
It seems that for both cases [Ondřej Surý](https://askubuntu.com/questions/1333214/can-i-trust-a-repo-i-am-unfamiliar-with-to-provide-me-a-secure-php-8-0-package) comes in.

Perhaps @pawlean and I can try creating a poll on our discord server to see if we get any input regarding this, thanks for the idea 🙏

Reply:
> Lastly, is this something you would still like to own? 🤔 If yes, what do you think of the new scripts to build and test chunks and combinations? 🙏

Yeah, happy to collab on this PR 🙏 
I think that it's much better now since we can save our time, however I also think that it would be nicer if we had one `dazzle` wrapper script for all tasks with subcommands baked in, instead of multiple `dazzle-*.sh` files 😄

Reply:
The intent of getting the scripts out there was to ship skateboards. If we find having many files is proves problematic, we can certainly iterate to condense in the future.